tidb-cloud/set-up-vpc-peering-connections.md

```diff
@@ -13,41 +13,38 @@ To connect your application to TiDB Cloud via VPC peering, you need to set up [V
 
 > **Tip:**
 >
 > To connect your application to TiDB Cloud, you can also set up [private endpoint connection](/tidb-cloud/set-up-private-endpoint-connections.md) with TiDB Cloud, which is secure and private, and does not expose your data to the public internet. It is recommended to use private endpoints over VPC peering connections.
 
-## Prerequisite: Set a Project CIDR
+## Prerequisite: Set a CIDR for this region
 
-Project CIDR (Classless Inter-Domain Routing) is the CIDR block used for network peering in a project.
+CIDR (Classless Inter-Domain Routing) is the CIDR block used for creating VPC for TiDB Dedicated Cluster .
 
-Before adding VPC Peering requests to a region, you need to set a project CIDR for your project's cloud provider (AWS or Google Cloud) to establish a peering link to your application's VPC.
+Before adding VPC Peering requests to a region, you need to set a CIDR for this region , and then create the first Dedicated Cluster in this region. After the first Dedicated Cluster is created, the VPC of the cluster is created, then you can establish a peering link to your application's VPC.
 
-You can set the project CIDR when creating the first TiDB Dedicated of your project. If you want to set the project CIDR before creating the cluster, perform the following operations:
+You can set the CIDR when creating the first Dedicated Cluster. If you want to set the CIDR before creating the cluster, perform the following operations:
 
 1. Log in to the [TiDB Cloud console](https://tidbcloud.com).
 2. Click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner, switch to the target project if you have multiple projects, and then click **Project Settings**.
 3. On the **Project Settings** page of your project, click **Network Access** in the left navigation pane, and then click the **Project CIDR** tab.
-4. Click **Add a project CIDR for AWS** or **Add a project CIDR for Google Cloud** according to your cloud provider, specify one of the following network addresses in the **Project CIDR** field, and then click **Confirm**.
+4. Click **Create CIDR** ,then click **AWS CIDR** or **Google Cloud CIDR** according to your cloud provider, specify the Region and CIDR value in the  **Create AWS CIDR** or **Create Google Cloud CIDR**  window , and then click **Confirm**.
 
     > **Note:**
     >
-    > To avoid any conflicts with the CIDR of the VPC where your application is located, you need to set a different project CIDR in this field.
+    > To avoid any conflicts with the CIDR of the VPC where your application is located, you need to set a different project CIDR in this field. TiDB Cloud recommend configuring an IP range size between /16 and /23. The following network addresses are supported:
```

Comment:
```suggestion
    > To avoid any conflicts with the CIDR of the VPC where your application is located, you need to set a different project CIDR in this field. It is recommended to configure an IP range size between /16 and /23. The following network addresses are supported:
```